config/routes.rb

```diff
@@ -18,7 +18,7 @@
     resources :roles, only: %i[index create update destroy]
   end
 
-  resources :workspaces, only: %i[index show destroy edit update] do
+  resources :workspaces, except: :new do
```

Comment:
Allow workspaces to be created outside of the project context (for duplicate use case).